scripts/run_build_pdo_snowflake.bat

```diff
@@ -0,0 +1,31 @@
+::
```

Comment:
This is necessary since the `phpsdk-starter.bat` file opens up a new shell so you have to tell the script to run a file as a task.